LinaTheExplorer/LinaTheExplorer/Adventures_don't_wait/WeatherResponse.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Net;
+
+namespace LinaTheExplorer
+{
+    public class WeatherResponse
+    {
+
+        public static int[] GetWeather(string city)
+        {
+            string url1 = $"https://65130ff8-40ce-4af3-b822-611ec2546736.mock.pstmn.io/weather/source_b?year=2022&month=07&city={city}";
```

Comment:
seems you mistyped URLs, here you use same sources (source b two times), but per the task you need to use source A and source B